tools/package-source.js

```diff
@@ -1712,6 +1711,37 @@ _.extend(PackageSource.prototype, {
     return _.keys(packages);
   },
 
+  // Returns an array of objects, representing this package's public
+  // exports. Each object has the following keys:
+  //  - name: export name (ex: "Accounts")
+  //  - arch: an array of strings representing architectures for which this
+  //    export is declared.
+  //
+  // This ignores testOnly exports.
+  getExports: function () {
+    var self = this;
+    var ret = {};
+    // Go over all of the architectures, and aggregate the exports together.
+    _.each(self.architectures, function (arch) {
+      _.each(arch.declaredExports, function (exp) {
+        // Skip testOnly exports -- the flag is intended for use in testing
```

Comment:
The big "just one symbol" change is just me thinking out loud, definitely not for this change.

Yeah, I think keeping this change as "we don't send testOnly to the server" and thinking about changes to testOnly separately makes sense.  And the answer might totally just be "take testOnly off of test-helpers, and change the definition of testOnly exports to be 'imported only by a package's own tests, not by any tests'".

(But longer term we should maybe just consider a default symbol, since it's actually a kind of annoying pattern to have to do "OK, which is the first file that gets evaluated, so I can put `MyPackageNameTest = {}` in it" since that really does only seem to be used for that purpose.)